build/docker/Dockerfile

```diff
@@ -13,6 +13,8 @@ LABEL description="Honeydipper - an event-driven orchestration framework" \
       org.label-schema.vcs-url=https://github.com/honeydipper/honeydipper \
       org.label-schema.schema-version="1.0"
 
+RUN apk update && apk add ca-certificates && rm -rf /var/cache/apk/*
```

Comment:
bikeshedding a little, but I think this can just be
```
 RUN apk add  --no-cache ca-certificates
```
Ran it by `hadolint` just to be sure:
```
build/docker/Dockerfile:16 DL3019 Use the `--no-cache` switch to avoid the need to use `--update` and remove `/var/cache/apk/*` when done installing packages
```
https://github.com/hadolint/hadolint/wiki/DL3019